packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuideButton/FieldGuideButton.js

```diff
@@ -41,20 +42,35 @@ export function ButtonLabel() {
         {t('FieldGuide.FieldGuideButton.buttonLabel')}
       </StyledSpacedText>
       {/** Same styling as ImageToolbar > Button */}
-      <HelpIcon
-        fill='white'
-        width='min(50%, 1.2rem)'
-      />
+      <HelpIcon fill='white' width='min(50%, 1.2rem)' />
     </Box>
   )
 }
 
-function FieldGuideButton({ fieldGuide = null, onClick = () => true, theme }) {
+function RowButtonLabel() {
+  const { t } = useTranslation('components')
+
+  return (
+    <Box direction='row' gap='8px' justify='center' align='center'>
```

Comment:
This is very minor, but the ColumnButtonLabel includes `as='span'`, not sure if we want to do the same with this label.

Reply:
Good catch! I'll update accordingly